src/electron/views/renderer-initializer.ts

```diff
@@ -410,6 +413,14 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch)
 
         scanController.initialize();
 
+        const deviceFocusControllerFactory = new DeviceFocusControllerFactory(
+            appiumAdbWrapperFactory,
+            createDeviceFocusCommandSender(axios.get),
+        );
+
+        // Placeholder--remove once we include deviceFocusControllerFactory in the deps
+        deviceFocusControllerFactory.initialize();
```

Comment:
You can actually just pass this into the deps as it is and can remove this and the initialize method. I don't think TS/lint would complain if there is an additional property on the deps object that is not needed.